plugins/sources/frontend/public/stylesheets/main.css

```diff
@@ -1,3 +1,19 @@
 .source-table tr{cursor: pointer;}
 .source-20{width: 20%;}
-.source-40{width: 40%;}
\ No newline at end of file
+.source-40{width: 40%;}
+
+.bu-is-6 .keywordsDashboardWidget .cly-vue-metric-cards .cly-vue-metric-card {
```

Comment:
A couple of things here:

-First I'm not sure if it is a good idea to limit the rule to bu-is-6. Any specific reasons for that?
-camelCase / kebab-case confusion here as well.
-.cly-vue-metric-cards and .cly-vue-metric-card being overridden is a small violation but I guess you had no choice, right?

All these apply to following rules